gcsfs/dask_link.py

```diff
@@ -121,4 +126,4 @@ def walk(self, path, refresh=False):
 try:
     register()
 except ImportError as e:
-    print(e)
+    logger.debug(e)
```

Comment:
should be a string

Reply:
Here is an implicit conversion like the current implementation (print(e)). Maybe no need to add str(e). I can change it if you insist.

Reply:
Seems you are correct, sorry for the noise.